packages/vector_graphics_compiler/lib/src/geometry/path.dart

```diff
@@ -0,0 +1,475 @@
+import 'dart:math' as math;
+
+import 'package:meta/meta.dart';
+import 'package:path_parsing/path_parsing.dart';
+
+import 'basic_types.dart';
+import 'matrix.dart';
+import '../util.dart';
+
+const double _kArcApproximationMagic = 0.551915024494;
```

Comment:
assuming you took this constant from somewhere, would be nice to cite

Reply:
Impeller. Will add a comment.